helpers/settings.go

```diff
@@ -138,7 +141,41 @@ func (s *Settings) InitSettings(envVars *EnvVars, env *cfenv.App) (retErr error)
 		return GenerateRandomString(32)
 	}
 
+	// Initialize CSRF key
+	var err error
+	// First, try the new env variable
+	s.CSRFKey, err = hex.DecodeString(envVars.String(CSRFKeyEnvVar, ""))
+	if err != nil {
+		return err
+	}
+
+	// Fall back to legacy key variable and format - consider printing deprecation warning
+	if len(s.CSRFKey) == 0 {
+		s.CSRFKey = []byte(envVars.String(LegacySessionKeyEnvVar, ""))
+	}
+
+	// Return error if not found
+	if len(s.CSRFKey) == 0 {
+		return &ErrMissingEnvVar{Name: CSRFKeyEnvVar}
+	}
+
 	// Initialize Sessions.
+	// First, try the new env variable
+	sessionAuthenticationKey, err := hex.DecodeString(envVars.String(SessionAuthenticationEnvVar, ""))
+	if err != nil {
+		return err
+	}
+
+	// Fall back to legacy key variable and format
+	if len(sessionAuthenticationKey) == 0 {
+		sessionAuthenticationKey = []byte(envVars.String(LegacySessionKeyEnvVar, ""))
```

Comment:
same thing here as https://github.com/18F/cg-dashboard/pull/1238/files#r142680218
But just switch  to SessionAuthenticationEnvVar

Reply:
Fixed.